src/main/java/com/synopsys/integration/alert/common/configuration/FieldAccessor.java

```diff
@@ -79,4 +79,11 @@ public FieldAccessor(final Map<String, ConfigurationFieldModel> fields) {
         return Optional.empty();
     }
 
+    public boolean isSet(final String key) {
+        if (StringUtils.isNotEmpty(key) && fields.containsKey(key)) {
```

Comment:
We probably want to use `StringUtils.isNotBlank(...)`. Blank accounts for all whitespace strings, empty only checks `null` or the empty string `""`.

Reply:
good catch I think we have isNotEmpty used in another spot in the class.  I will change that as well